classes/i18n/LocaleConversion.inc.php

```diff
@@ -0,0 +1,229 @@
+<?php
+declare(strict_types = 1);
+
+/**
+ * @defgroup i18n I18N
+ * Implements localization concerns such as locale files, time zones, and country lists.
+ */
+
+/**
+ * @file classes/i18n/LocaleConversion.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class LocaleConversion
+ * @ingroup i18n
+ *
+ * @brief Provides methods to convert locales from one format to another
+ */
+
+namespace PKP\i18n;
+
+use PKP\facades\Locale;
+
+class LocaleConversion
```

Comment:
I found a package to address these conversions, but as we have some "business logic" here, I just had it moved to a separate class.